client/state/data-layer/wpcom-http/index.js

```diff
@@ -45,18 +54,32 @@ const queueRequest = ( { dispatch }, action, next ) => {
 		{ path, formData },
 		query,
 		method === 'POST' && body,
-		( error, data ) => !! error
-			? onFailure && dispatch( extendAction( onFailure, failureMeta( error ) ) )
-			: onSuccess && dispatch( extendAction( onSuccess, successMeta( data ) ) )
+		( error, data ) => {
+			const {
+				failures,
```

Comment:
I disagree strongly here on this point. the kinds of processors we have talked about would almost certainly rely on this behavior unless we introduce code we know will produce bugs. the dedupe processor is one example and another could be in batching requests.

imagine, for example, that we batch `GET` requests into the batch endpoint. there could be tens of requests sharing the same network response and when that response comes back in we would _have to_ be able to fan out the handlers.